### ISSUE TITLE
FE-JS:Transaction Handling

### ISSUE THREAD

**Issue by NadigerAmit** [mlabs-haskell/TuxedoDapp#21]
Need to be handled in JS app also : https://github.com/mlabs-haskell/TuxedoDapp/issues/6

**Comment by AltiMario** [comment]
@philoniare can we consider this completed?

**Comment by philoniare** [comment]
@AltiMario no, we don't keep track of transaction status. As far as I know, we just call the API to initiate the transaction. So, we don't have this part `Provide feedback to users on transaction status and confirmations.`

**Comment by AltiMario** [comment]
ok it is something to have with a low priority
